veloroutes_voies_vertes/test/test_data.py

```diff
@@ -21,6 +23,23 @@ def test_trigger_repere_numero_serie(self):
             self.cursor.execute(
                 "INSERT INTO veloroutes.repere(type_noeud) VALUES ('CPT')")
 
+    def test_insertok_numero_serie(self):
+        """Test that we can insert a row in repere with good items"""
+        self.cursor.execute("TRUNCATE TABLE veloroutes.repere CASCADE")
+        sql = """
+            INSERT INTO veloroutes.repere(libelle,numero_serie,type_noeud,geom)
+            VALUES ('libellé test', 'num_serie_test', 'DFE',
+                    '01010000206A080000A84C000037B72C41708D9999F2005A41')
```

Comment:
```suggestion
                    ST_GeomFromText('POINT(0 0)'))
```
Même si ce n'est pas le but du test d'avoir une géométrie, c'est plus lisible d'avoir une géométrie lisible.

Reply:
je suis d'accord, je dois juste rajouter le SRID mais c'est vrai que c'est mieux!